Phonebook.TwilightSaw/Controller/AppDbContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using TwilightSaw.Phonebook.Model;
+
+namespace TwilightSaw.Phonebook.Controller;
+
+public class AppDbContext : DbContext
+{
+    public DbSet<Contact> Contacts { get; set; }
+
+    public DbSet<Category> Categories { get; set; }
+
+    private readonly IConfiguration _configuration;
+
+    public AppDbContext(DbContextOptions<AppDbContext> options, IConfiguration configuration)
+        : base(options)
+    {
+        _configuration = configuration;
+    }
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        var connectionString = _configuration.GetConnectionString("DefaultConnection");
+        optionsBuilder.UseSqlServer(connectionString).UseLazyLoadingProxies();
+        optionsBuilder.UseSqlServer(connectionString).LogTo(Console.WriteLine, LogLevel.None);
```

Comment:
🟡 **LogTo(Console.WriteLine, LogLevel.None)**

❓ Personal query... what is your intention around this line?